katello/hooks/pre_validations/31-upgrade-puppet.rb

```diff
@@ -1,6 +1,6 @@
 unless app_value(:noop)
-  if File.exist?('/opt/puppetlabs/puppet/bin/ruby') && execute("rpm -q puppet-agent-oauth", false, false)
-    unless execute("/opt/puppetlabs/puppet/bin/ruby -e \"require 'oauth'\"", false, false)
+  if File.exist?('/opt/puppetlabs/puppet/bin/ruby') && system("rpm -q puppet-agent-oauth")
+    unless system("/opt/puppetlabs/puppet/bin/ruby -e \"require 'oauth'\"")
```

Comment:
These two are moving to system since we want to know if it's successful but without logging or stopping the execution.